charts/move2kube/templates/01-move2kube-instance.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.instance.name }}
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.instance.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.instance.name }}
+    spec:
+      containers:
+        - name: {{ .Values.instance.name }}
+          image: {{ .Values.instance.image }}
+          securityContext:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+          ports:
+            - containerPort: 8080
+          env:
+            - name: SSH_AUTH_SOCK
+              value: /tmp/unix-socket
+          volumeMounts:
+            - name: ssh-priv-key
+              readOnly: true
+              mountPath: "/root/.ssh/id_rsa"
+              subPath: id_rsa
+            - name: ssh-pub-key
+              readOnly: true
+              mountPath: "/root/.ssh/id_rsa.pub"
+              subPath: id_rsa.pub
+          lifecycle:
+            postStart:
+              exec:
+                command: [ "/bin/sh", "-c", "ssh-agent -a /tmp/unix-socket && ssh-add /root/.ssh/id_rsa" ]
+      volumes:
+        - name: ssh-priv-key
+          secret:
+            secretName: {{ .Values.sshSecretName }}
```

Comment:
I can't see in this runbook when the Secret named as `.Values.sshSecretName` is generated. Am I missing anything?

Reply:
No, I missed it!